README.md

```diff
@@ -420,6 +420,31 @@ $ friends add nickname "Grace Hopper" "Amazing Grace"
 Nickname added: "Grace Hopper (a.k.a. The Admiral a.k.a. Amazing Grace)"
 ```
 
+#### Add a default location
```

Comment:
I have a slight preference for "Setting a default location" but am open to being convinced otherwise! Thoughts?

Reply:
Your suggestion is waaay more descriptive, I'm with you :) 👍